### ISSUE TITLE
Support image embeds in wysiwyg editor

### ISSUE THREAD

**Issue by luxaritas** [eternagame/eternagame.org#365]
### Describe your idea

You should be able to add an embedded image in the wysiwyg editor (with alt text!)

### Motivation

In a number of areas, users may prefer not having to have the full URL listed, but instead add a link to some existing text

### Proposed Solution

New wysiwyg editor button (+ keyboard shortcut?)

### Other Information

For now, this should only support existing image URLs, though in the future we may support image uploads
